### ISSUE TITLE
Core dump with small fragment bin size

### ISSUE THREAD

**Issue by wsnoble** [crux-toolkit/crux-toolkit#556]
I am getting a core dump from this command:

```
crux tide-search \
    --auto-mz-bin-width warn \
    Adult_Adrenalgland_bRP_Velos_1_f01.ms2.gz human.isoforms
```

The problem goes away if I turn off the auto-mz-bin-width.  Here is the command I used to generate the index:

```
crux tide-index --output-dir human.isoforms \
        --peptide-list T \
        human.isoforms.fasta human.isoforms
```

The relevant part of the log file is

```
INFO: Starting search.
INFO: 1000 spectrum-charge combinations searched, 31% complete
INFO: 2000 spectrum-charge combinations searched, 63% complete
INFO: 3000 spectrum-charge combinations searched, 94% complete
INFO: [Thread 0]: Deleted 0 precursor, 0 isotope and 0 out-of-range peaks.
INFO: [Thread 0]: Retained 100% of peaks.
*** Error in `/net/noble/vol1/home/noble/proj/crux/crux-toolkit/build-fix-bug-in-tide-search-speedup/src/crux': free(): invalid pointer: 0x000000001d44e8f0 ***
```

**Comment by kfattila** [comment]
Did you try it with pull request for the bug in issue #554?

**Comment by kfattila** [comment]
A minor issue:
```
(base) attila@L429746:~/crux-toolkit-02-07/test/performance-tests$ ./runall
worm+contaminants/tide-index.peptides.txt exists.
RUN: ../../src/crux tide-search --output-dir tide-xcorr --parameter-file crux.param  051708-worm-ASMS-10.ms2 worm+contaminants
WARNING: Read parameter '--auto-mz-bin-width' from parameter file, but no such parameter exists.
INFO: Writing results to output directory 'tide-xcorr'.

```

**Comment by kfattila** [comment]
I added a fix for this bug, it is in the pull request ```fixed a bug in tide-search speedup #554```

**Comment by wsnoble** [comment]
Yes, this fixed the problem. Thanks!  I'll resolve this on the assumption that it will get merged when the other branch is merged.
